bigchaindb/common/schema/transaction.yaml

```diff
@@ -0,0 +1,245 @@
+---
```

Comment:
Needs the `id` and `$schema` properties from https://github.com/bigchaindb/bigchaindb/pull/311/files#diff-1f1becfa0fbd3b2647c0b02c4fe99ebeR2.